buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```diff
@@ -227,6 +226,26 @@ pipeline {
         // Single quotes are required for token variables to avoid them leaking
         sh "${CISCRIPT_DIR}/publish-to-anaconda ${WORKSPACE}" + ' ${ANACONDA_TOKEN} ' +\
           "${ANACONDA_CHANNEL} ${ANACONDA_CHANNEL_LABEL} ${WORKSPACE}/conda-packages/*.tar.bz2"
+      }
+    }
+
+    stage ('Publishing to Github') {
+      when {
+        beforeAgent true
+        beforeOptions true
+        expression { env.PUBLISH_PACKAGES == 'true' }
```

Comment:
Thanks for this. It's a useful update.

The default behaviour for multiple expressions is for all to need to be true (https://www.jenkins.io/doc/book/pipeline/syntax/#when) so I assume that's what was meant? Could we make clear here that it requires that with an `allOf` block?